config/routes.rb

```diff
@@ -7,7 +7,7 @@
     root "high_voltage/pages#show", id: "landing"
   end
 
-  resources :lists, only: [:index, :create, :show] do
+  resources :lists, only: [:index, :create, :edit, :show, :update, :destroy] do
```

Comment:
Would you mind using these in "standard" order? `:index, :show, :new, :create, :edit, :update, :destroy`